### ISSUE TITLE
Tabbed plotting in summaries

### ISSUE THREAD

**Issue by justinbois** [justinbois/fish-activity#11]
It would be nice to have a complete family of plots with a tabbed plot. This behaves more or less like tabs in your browser, so you can switch from one plot to another.

**Comment by grigorios-oikonomou** [comment]
Yes! I assume you will have a core set of graphs that are generated automatically when some one runs the fishviz function?
